nengo_viz/static/viz_pan.js

```diff
@@ -0,0 +1,57 @@
+/* Calculates drag deltas on background */
+
+VIZ.pan = {};
+
+VIZ.pan.enabled = false;
+
+/*A Posn is an object with:
+ posn.x - int 
+ posn.y - int
+*/
+
+//Used for storing the cumulative x and y panning of the model
+VIZ.pan.cposn = {x:0, y:0};
+
+function init_main_events() {
```

Comment:
@cchan987 
There's something in javascript called self-invoking functions, or anonymous functions. Essentially, as soon as you define the function, it'll call itself right afterwards. Might make things a bit cleaner here.
http://www.w3schools.com/js/js_function_definition.asp